### PR TITLE
Update audits table for v0.8

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

File: src/components/audit/filters.vue [hunk 1]
```diff
@@ -68,6 +68,9 @@ export default {
         categoryOption('Form Actions', 'form'),
         actionOption('Create', 'form.create'),
         actionOption('Update Details', 'form.update'),
+        actionOption('Set Draft', 'form.update.draft.set'),
```
Comment (lognaturel) [inline src/components/audit/filters.vue]: I mentioned to @matthew-white that I wouldn't really know what "set" means here. He described that when a draft is first created there would be two of these logged: creating the draft and then setting the draft form definition. I think it's pretty confusing that there are two of the same event. I don't think this is hugely important and I recognize that this is what the backend does but wanted to flag it. CC @issa-tseng

Comment (issa-tseng) [reply src/components/audit/filters.vue]: what do you propose?

Comment (matthew-white) [reply src/components/audit/filters.vue]: It's a little verbose, but would "Create or Update Draft" be clearer than "Set Draft"?

Comment (issa-tseng) [reply src/components/audit/filters.vue]: sure

Comment (lognaturel) [reply src/components/audit/filters.vue]: Yes, that sounds great to me.

**Pull request merged**